### ISSUE TITLE
Reshaping array size problem

### ISSUE THREAD

**Issue by Zark02** [keras-team/keras#11475]
I am trying to run this code, i am giving 19 images as an input and 19 images as an output.
i am getting this error, "ValueError:Cannot reshape array of size 5377456 into shape (19,266,266,1)".
This is my code

import matplotlib.pyplot as plt
from os import listdir
from os.path import isfile, join
import numpy as np
import cv2
from PIL import Image
from scipy.fftpack import dct, idct
import scipy.fftpack

mypath='C:\\Users\\Zark\\Desktop\\thesis images\\input'
onlyfiles = [ f for f in listdir(mypath) if isfile(join(mypath,f)) ]
images = np.zeros((len(onlyfiles),266,266,4))
for n in range(0, len(onlyfiles)):
  images[n] = Image.open( join(mypath,onlyfiles[n]) )

  
mypath1='C:\\Users\\Zark\\Desktop\\thesis images\\output'
onlyfiles1 = [ f for f in listdir(mypath1) if isfile(join(mypath1,f)) ]
images1 = np.zeros((len(onlyfiles1),266,266,4))
for k in range(0, len(onlyfiles1)):
  myimage = Image.open( join(mypath1,onlyfiles1[k]) )
#plt.imshow(images[0])

##img = cv2.imread(images[0], 0)      # 1 chan, grayscale!
#dct_arr = np.zeros((images.shape[0],180,180))
#for k in range(images.shape[0]):
#    dct_arr[k] = dct2(images[k])
    
#idct = idct2(dct)
#
#plt.imshow(idct)
    
    
import keras
#from keras.datasets import mnist
from keras.models import Sequential
from keras.layers import Dense, Dropout, Flatten
from keras.layers import Conv2D, MaxPooling2D
from keras import backend as K

batch_size = 64
#num_classes = 10
epochs = 20

# input image dimensions
img_rows, img_cols = 266, 266

# the data, split between train and test sets
#(x_train, y_train), (x_test, y_test) = mnist.load_data()
x_train =  images
y_train =  images1

if K.image_data_format() == 'channels_first':
    x_train = x_train.reshape(x_train.shape[0], 1, img_rows, img_cols)
    y_train = y_train.reshape(y_train.shape[0], 1, img_rows, img_cols)
    #x_test = x_test.reshape(x_test.shape[0], 1, img_rows, img_cols)
    input_shape = (1, img_rows, img_cols)
else:
    y_train = y_train.reshape(y_train.shape[0], img_rows, img_cols, 1)
    #x_test = x_test.reshape(x_test.shape[0], img_rows, img_cols, 1)
    input_shape = (img_rows, img_cols, 1)

x_train = x_train.astype('float64')
y_train = y_train.astype('float64')
#x_test = x_test.astype('float32')
x_train /= 255
y_train /= 255
#x_test /= 255
print('x_train shape:', x_train.shape)
print(x_train.shape[0], 'train samples')

print('y_train shape:', y_train.shape)
print(y_train.shape[0], 'train samples')
#print(x_test.shape[0], 'test samples')

# convert class vectors to binary class matrices
#y_train = keras.utils.to_categorical(y_train, num_classes)
#y_test = keras.utils.to_categorical(y_test, num_classes)

model = Sequential()
model.add(Conv2D(16, kernel_size=(3, 3), padding='same',
                 activation='relu',
                 input_shape=input_shape))
#model.add(ZeroPadding2D(padding=(1, 1))
model.add(Conv2D(32, (3, 3), padding='same', activation='relu'))
#model.add(Conv2D(64, (3, 3), activation='relu'))
#model.add(MaxPooling2D(pool_size=(2, 2)))
#model.add(Dropout(0.25))
#model.add(Flatten())
#model.add(Dense(128, activation='relu'))
#model.add(Dropout(0.5))
model.add(Conv2D(1, (3, 3 ), padding='same'))

model.compile(loss=keras.losses.mean_squared_error,
              optimizer=keras.optimizers.Adadelta())
                #,metrics=['loss'])

model.fit(x_train, y_train,
          batch_size=batch_size,
          epochs=epochs,
          verbose=1)

**Comment by farizrahman4u** [comment]
Please format your code.
